src/audioVoice.js

```diff
@@ -43,6 +43,11 @@ class AudioVoice {
     this.output.disconnect();
   }
 
+  /**
+   * Dispose the output if it exists
```

Comment:
For this particular class, it disconnects the output node, but in general, users should call `dispose` on a node when they want to fully disconnect it and will never use it again. Do you think we can make the description also mention that so it's clearer when and why one should call `dispose` instead of just `disconnect`?

Reply:
I'm imagining that, since the distinction between `dispose` and `disconnect` should apply to (I think?) all nodes, once we have some text for it in this class, we can reuse that text in the docs for `dispose` in other classes too. Actually adding it to other classes can happen in separate PRs of course, but that's my line of thinking when I'm reviewing the docs here 🙂

Reply:
Yes, I have noticed some documentation in the monosynth and polysynth for `dispose()` which says `Get rid of the MonoSynth and free up its resources / memory.`